src/xdocs/config_blocks.xml

```diff
@@ -1045,6 +1045,26 @@ allowedFuture.addCallback(result -> {
 &lt;module name=&quot;RightCurly&quot;/&gt;
         </source>
 
+        <p>Example: </p>
+        <div class="wrapper">
+          <pre>
```

Comment:
xdoc examples and config should be surrounded by `source` tag and not `div` and `pre`.